ts/src/renderer/phaser/classes/PhaserUnit.ts

```diff
@@ -92,25 +118,24 @@ class PhaserUnit extends PhaserAnimatedEntity {
 
 		label.setText(data.text || '');
 
-		label.y = -25 -
-					Math.max(this.sprite.displayHeight, this.sprite.displayWidth) / 2;
-		label.setScale(1.25);
+		const sprite = this.sprite;
+		label.y = -25 - (sprite.displayHeight + sprite.displayWidth) / 4;
```

Comment:
it might be good also to make methods for offsetting attributes and label so that logic can be reused as well